src/crypto/deviceinfo.ts

```diff
@@ -25,7 +25,7 @@ export interface IDevice {
     signatures?: ISignatures;
 }
 
-enum DeviceVerification {
+export enum DeviceVerification {
```

Comment:
suggest moving this into `models/device.ts` while you're here. I want to kill off the whole `crypto` directory eventually.